cluster.go

```diff
@@ -374,11 +401,120 @@ func (cfg *ClusterConfig) Validate() error {
 		return errors.New("ReconnectionPolicy.GetMaxRetries returns non-positive number")
 	}
 
-	return nil
+	if cfg.PageSize < 0 {
```

Comment:
This comment is about all this checks, if we want to accept positive numbers (>0) then the check should be <=0, but if 0 is ok then we should call it non-negative numbers not positive in the error message

Reply:
Sure, let's change working there.

Reply:
Please take a look